### PR TITLE
Add logstash_plugin to manage logstash plugins

### PR DESCRIPTION
**Pull request by nerzhul**
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
logstash_plugin

##### ANSIBLE VERSION
```
2.2.1.0
```

##### SUMMARY

Add support to logstash_plugin module, inspired by elasticsearch_plugin module. This module permits to manage logstash plugins (add, remove, specific version)

### CODE REVIEW

File: lib/ansible/modules/monitoring/logstash_plugin.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+module: logstash_plugin
+short_description: Manage Logstash plugins
+description:
+    - Manages Logstash plugins.
+version_added: "2.3"
+author: Loic Blot (@nerzhul)
+options:
+    name:
+        description:
+            - Name of the plugin to install.
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/logstash_plugin.py]: All `description:` (apart from `short_description:`) must be full sentences (capital letters & fullstops)

File: lib/ansible/modules/monitoring/logstash_plugin.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    version:
+        description:
+            - Version of the plugin to be installed.
+              If plugin exists with previous version, it will NOT be updated
+        required: False
+        default: None
+'''
+
+EXAMPLES = '''
+# Install Logstash beats input plugin
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/logstash_plugin.py]: Please write as
```
- name: Install Logstash beats input plugin
  logstash_plugin:
```

File: lib/ansible/modules/monitoring/logstash_plugin.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    name: logstash-input-syslog
+    version: '3.2.0'
+
+# Uninstall Logstash plugin
+- logstash_plugin:
+    state: absent
+    name: logstash-filter-multiline
+'''
+
+from ansible.module_utils.basic import *
```
Comment (gundalow) [inline lib/ansible/modules/monitoring/logstash_plugin.py]: Please only import what you need, and not `*`

**Review by gundalow — CHANGES_REQUESTED** [review]
Please update `CHANGELOG.md` to add the new module into the 2.3 release section.

**Comment by nerzhul** [comment]
ready_for_review

**Review by gundalow — APPROVED** [review]

**Comment by nerzhul** [comment]
thanks @gundalow

**Pull request merged**